rero_ils/modules/acq_invoices/mappings/v7/acq_invoices/acq_invoice-v0.0.1.json

```diff
@@ -1,6 +1,6 @@
 {
   "settings": {
-    "number_of_shards": 1,
```

Comment:
Probably we can use an es index template: https://www.elastic.co/guide/en/elasticsearch/reference/7.10/index-templates.html

Reply:
+1

Reply:
We will do this in a separate PR.